test/src/main/scala/scala/test/scala_import/BUILD

```diff
@@ -4,7 +4,7 @@ load("//scala:scala_import.bzl", "scala_import")
 #Many jars
 scala_import(
   name = "guava_and_commons_lang",
-  jars = ["@com_google_guava_guava_21_0//jar:file", "@org_apache_commons_commons_lang_3_5//jar:file"],
+  jars = ["@com_google_guava_guava_21_0//:file", "@org_apache_commons_commons_lang_3_5//jar:file"],
```

Comment:
@ittaiz re-posting my question right next to the example of the consumer:

currently java_import_external does not support depending on the jar file directly. in order to not do major changes to justine's code, I've had to change the usage label structure from `@com_google_guava_guava_21_0//jar:file` to `@com_google_guava_guava_21_0//:file`.
Is this legitimate or should I re-write the code to have the same folder structure as `maven_jar`?

Alternatively, you've stated that `scala_import` can depend on full jar files without needing the `file` filegroup target.

so with the example above, I can just change it to 
jars = ["@com_google_guava_guava_21_0//jar", "@org_apache_commons_commons_lang_3_5//jar"],

and that will solve the issue (and still not take ijars)?